vendor/app.css

```diff
@@ -0,0 +1,105 @@
+:root {
```

Comment:
I still feel uncomfortable with all this css. I fear the side effects on the apps using the banner. Didn't you find anything you needed via https://appuniversum.github.io/appuniversum/ ?